packages/docs/src/routes/docs/faq/index.mdx

```diff
@@ -209,3 +210,13 @@ An amazing team of contributors around the world living in [Discord](https://qwi
 ## Is Qwik open source?
 
 Yes, [MIT](https://github.com/BuilderIO/qwik/blob/main/LICENSE) and [dependency-free](https://github.com/BuilderIO/qwik/blob/main/packages/qwik/package.json), installing Qwik will not bloat your node_modules nor your lawyers.
+
+# Does Qwik have any downside?
+
+Yes. Every framework, possessing its own strengths and weaknesses, involves a trade off.
+
+1. As a relative newcomer to the JS world, Qwik has yet to develop a community of libraries and best practices.
+2. Qwik’s magic lies in its serialization, but this also brings about some limitations developers wouldn’t otherwise have to consider.
+3. Qwik’s core idea is lazy-loading as much as possible, which sometimes requires developers to restart the dev server to preview changes.
```

Comment:
ah yes this sounds like a bug - @TejasQ hit this too it sounded like from his great youtube vid

Reply:
OK I'll remove this line!